test/e2e/tests.ts

```diff
@@ -275,11 +291,16 @@ describe("authorization check", () => {
 
 describe("broadcast changes", () => {
   const table = "broadcast_changes";
-  const id = 1;
+  const id = crypto.randomUUID();
 
   it("authenticated user receives insert broadcast change from a specific topic based on id", async () => {
     let supabase = await createClient(url, token, { realtime });
-    let accessToken = await signInUser(supabase, "test1@test.com", "test_test");
+    let accessToken = await signInUser(
+      supabase,
+      "filipe@supabase.io",
```

Comment:
/nit Not sure what the realtime stance is but in Auth we generally exclude references to @supabsae.io

Can consider changing the test email address

Reply:
we need it for Supabase staging projects 😓